source/synthDrivers/espeak.py

```diff
@@ -193,9 +193,11 @@ def _getAvailableVoices(self):
 		voices=OrderedDict()
 		for v in _espeak.getVoiceList():
 			l=v.languages[1:]
+			# #7167: Some languages names require unicode characters EG: Norwegian Bokmål

```

Comment:
nit: I think this would be more readable as "Some language names contain Unicode characters".